Added ISIC Archive, CaPTk (#27)
jhudsl/ITCR_Tables: alldataFilesEditor.Rmd

## Edit 1: Add a column "identifier" to the resource table to use it as a merge key variable with the Data provided column entries in other tables
## Edit 2: Make sure add new identifiers for new resources in the resource data set.
resource_identifier <- new_resource %>% 
  mutate(identifier = c("PDX", "HemOnc", "ngchm", "OncoMX", "MethCon5", "CIViC", "BioMuta", "TCIA", "SCGV", "ivyGlimpse", "TCPA", "dcmqi", "Region Templates", "Cistrome", "TOPAS", "TCGA", "1000 Genomes", "TARGET", "CCLE", "dbGAP", "CPTAC", "PheKB", "NSCLC Radiomics", "ICGC", "ISIC"))  %>%  relocate(identifier)

## Save the edited data file
write.csv(resource_identifier, file = "data/ictr_table_resources_identifier.csv")
```

## Adding new platforms to the Imaging Table

```{r,echo=FALSE,results='hide',message=FALSE, warning=FALSE}
# imaging <- read_csv(file = here::here("data/ictr_table_imaging.csv"))
# nameOfCols <- imaging %>% colnames()

## CaPTk: Submitted 06/01/2023, 11:12 (sbakas@upenn.edu)
new_imaging_CaPTk <- tibble(
  Name = "The Cancer Imaging Phenomics Toolkit (CaPTk)",
  `Link (Hyperlinked Over Name)` = "https://www.med.upenn.edu/cbica/captk/",
  Subcategories = "Radiomics", 

>   `Unique Factor` = "CaPTk beyond being a stand-alone software client it also makes its computational functionality on image analytics available for free, through the Image Processing Portal (IPP) that offers compute power through the CaPTk's supporting high performance computing cluster.",

Reviewer: ```suggestion
  `Unique Factor` = "CaPTk is a stand-alone software client that allows for free image analysis through the Image Processing Portal (IPP) that offers compute power through CaPTk's supporting high performance computing cluster.",
```

Reviewer: just simplifying a bit
